app/src/main/cpp/skyline/applet/swkbd/software_keyboard_applet.cpp

```diff
@@ -69,11 +69,16 @@ namespace skyline::applet::swkbd {
                   std::move(onNormalDataPushFromApplet),
                   std::move(onInteractiveDataPushFromApplet),
                   appletMode} {
-        if (appletMode != service::applet::LibraryAppletMode::AllForeground)
-            throw exception("Inline Software Keyboard not implemeted");
+        mode = appletMode;
     }
 
     Result SoftwareKeyboardApplet::Start() {
+        if (mode != service::applet::LibraryAppletMode::AllForeground) {
+            Logger::Warn("Stubbing out InlineKeyboard!");
```

Comment:
'Inline swkbd is implemented'

Reply:
I genuinely don't understand what do you mean by this comment

Reply:
replace the string you already had ("Stubbing out InlineKeyboard!") with 'Inline swkbd is implemented'

Reply:
@Maccraft123 Billy made a typo; he meant `Inline swkbd is unimplemented`, this fits our log style more closely.